Misc/NEWS.d/next/Library/2021-12-22-12-02-27.bpo-20392.CLAFIp.rst

```diff
@@ -0,0 +1 @@
+Fix inconsistency with uppercase file extensions in :meth:`MimeTypes.guess_type`. Patch by Kumar Aditya.
```

Comment:
I think it is safe to say that the original patch was authored by @rodriggo.parra

Link: https://bugs.python.org/file33703/case_guess_type.patch

Reply:
I didn't even read the patch as it was 7 years old and outdated.